specification/network/resource-manager/Microsoft.Network/stable/2018-04-01/applicationSecurityGroup.json

```diff
@@ -0,0 +1,319 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NetworkManagementClient",
+    "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+    "version": "2018-02-01"
```

Comment:
Invalid ApiVersion for this folder. Same pattern applies to all files that are currently 2018-02-01 if necessary (don't touch VMSS files)